internal/serve/httphandler/login_handler_test.go

```diff
@@ -71,6 +90,17 @@ func Test_LoginHandler(t *testing.T) {
 	}
 
 	const url = "/login"
+	const email = "testuser@email.com"
+	const password = "pass1234"
+	const defaultReCAPTCHAToken = "XyZ"
+	defaultReqBody := fmt.Sprintf(
+		`{"email": "%s", "password": "%s", "recaptcha_token": "%s"}`, email, password, defaultReCAPTCHAToken)
```

Comment:
I think the naming here can be improved for better clarity. I mentioned this in a previous review on this same PR, but what does default mean vs non-default? If this is meant to be a login request body with or without recaptcha, you should improve make sure the const tame conveys that meaning.